src/screens/FeedScreen/index.js

```diff
@@ -140,7 +140,8 @@ const FeedScreen = (props) => {
       data: item,
       isCaching: true,
       haveSeeMore,
-      refreshParent: () => refreshMoreText(index, haveSeeMore)
+      refreshParent: () => refreshMoreText(index, haveSeeMore),
+      fromFeeds: true
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Adding `fromFeeds: true` to the object passed to the function call seems fine. It's a good way to pass additional context information. However, make sure that the receiving function can handle this new property correctly without causing any side effects.
```diff
      refreshParent: () => refreshMoreText(index, haveSeeMore),
+     fromFeeds: true
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->